crates/eww/src/widgets/widget_definitions.rs

```diff
@@ -1062,8 +1067,6 @@ fn build_systray(bargs: &mut BuilderArgs) -> Result<gtk::MenuBar> {
                 props.icon_size(icon_size);
             }
         },
-        // @prop pack-direction - how to arrange tray items
-        prop(pack_direction: as_string) { gtk_widget.set_pack_direction(parse_packdirection(&pack_direction)?); },
```

Comment:
I'm sorry to ask, but has this been removed on purpose?

Reply:
The `orientation` property partially supersedes it, allowing you to switch between horizontal and vertical layouts. The `ltr/rtl` provided by `pack_direction` isn't meaningful because the icons are already randomly ordered, and vice versa for `ttb/btt`.

Reply:
> The ltr/rtl provided by pack_direction isn't meaningful because the icons are already randomly ordered

At least for me, the implementation of @ralismark shows the icons in the same order that programs are adding them. For example I configure `:pack_directon "rtl"` and have three programs currently open:

```
[a]  [b]  [c]
```

If I launch a 4th program `[D]`, the icon is added to the left side:

```
[D]  [a]  [b]  [c]
```

which is nice if the tray is on the right side of the bar because then all previous icons stay in the same place and do not move around as new apps are launched.

Without this property, the icons always get added to the right side. They are only ordered randomly when the bar reloads (due to a config change for example) but I'd say this is not the normal case and I guess this is nothing that can be worked around easily.

Reply:
This is easy to implement, just need to choose between `pack_start()` and `pack_end()` when adding the icon widget to the container, and that would require adding another property for the user to configure.

Reply:
There is now a `prepend-new` property to achieve this goal.